### ISSUE TITLE
Laravel application not showing 

### ISSUE THREAD

**Issue by arjunv615** [NativePHP/laravel#352]
### What were you trying to do?

While starting the development server with the command php artisan native:serve, the Laravel application is not appearing in a native desktop window.

### What happened?

 It should now show my Laravel application running in a native desktop window.

### How to reproduce the bug

After completing your installation process mentioned in your docsWhile starting the development server with the command php artisan native:serve, the Laravel application is not appearing in a native desktop window.

### Package Versions

"version": "0.7.0"

### PHP Version

8.3.8

### Laravel Version

10.0.0 and in 11.9

### Node Version

22.5

### Which operating systems have you seen this occur on?

Windows

### OS version

 10.0.22631.4037

**Comment by insidert** [comment]
Did you install the dependencies first? ```php artisan native:install```

**Comment by arjunv615** [comment]
Yes, I have installed the dependencies using this command and followed all the steps outlined in the NativePHP documentation: [Getting Started - Installation](https://nativephp.com/docs/1/getting-started/installation)

**Comment by simonhamp** [comment]
@arjunv615 this can happen if there's an exception during the bootup sequence of Laravel. Please [check your logs](https://nativephp.com/docs/1/getting-started/debugging#check-the-logs) to see if there are any errors
